docs/grid/field_types.md

```diff
@@ -292,14 +291,14 @@ final class UserGrid extends AbstractGrid implements ResourceAwareGridInterface
 </details>
 
 In the `@Grid/Column/_prettyName.html.twig` template, you just need to
-render the value for example as you see below:
+render the value. For example:
 
 ```twig
 <strong>{{ data }}</strong>
 ```
 
-If you wish to render more complex grid fields just redefine the path of
-the field to root – `path: .` in the yaml and you can access all
+If you wish to render more complex grid fields, just redefine the path of
+the field to root – `path: .` in the yaml file and you can access all
 attributes of the object instance:
```

Comment:
yaml mention can be skipped, I think it should be same for php config file as well.

```suggestion
If you wish to render more complex grid fields, just redefine the path of
the field to root – `path: .` and then you can access all
attributes of the object instance:
```

Reply:
Done :)